lib/features/push_notification/presentation/listener/email_change_listener.dart

```diff
@@ -173,15 +173,17 @@ class EmailChangeListener extends ChangeListener {
     }
   }
 
-  void _showLocalNotification(PresentationEmail presentationEmail) {
-    final notificationPayload = NotificationPayload(emailId: presentationEmail.id);
-    log('EmailChangeListener::_showLocalNotification():notificationPayload: $notificationPayload');
-    LocalNotificationManager.instance.showPushNotification(
+  Future<void> _showLocalNotification(
+    UserName userName,
+    PresentationEmail presentationEmail
+  ) async {
+    await LocalNotificationManager.instance.showPushNotification(
```

Comment:
what is the different for this case with `await` and without `await`?

Reply:
This is just to ensure notifications are displayed one after another according to the email list